app/routes/map-feature/zoning-map-index.js

```diff
@@ -0,0 +1,31 @@
+import Route from '@ember/routing/route';
+
+export default class ZoningMapIndexRoute extends Route {
+  model(params) {
+    const { id } = params;
+    if (id === '30d-Subplan') {
+      return {
+        id,
+        title: 'SUB PLAN OF 30D',
+        zoningMapURL:
+          'https://nyc3.digitaloceanspaces.com/zoningmap/map30d_subplan.pdf',
+        historicalMapURL:
+          'https://nyc3.digitaloceanspaces.com/zoningmap/maps30d.pdf',
+      };
+    }
+    if (id.length === 2) {
+      return {
+        id,
+        title: id,
+        zoningMapURL: `https://nyc3.digitaloceanspaces.com/zoningmap/map0${id}.pdf`,
+        historicalMapURL: `https://nyc3.digitaloceanspaces.com/zoningmap/maps0${id}.pdf`,
+      };
+    }
+    return {
```

Comment:
Users can place arbitrary ids in the url. How does the application behave when a user inputs an invalid id?

Reply:
You get the side panel with an error message which, while not ideal, is consistent with what other selectable layers do
<img width="1477" alt="image" src="https://github.com/user-attachments/assets/6bbd81f0-435f-42dd-bbd2-1d407752c80d" />